.github/workflows/label-check.yml

```diff
@@ -0,0 +1,16 @@
+name: label-check
+
+on:
+  pull_request_target:
+    types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  label-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: jesusvasquez333/verify-pr-label-action@v1.4.0
```

Comment:
have we done due diligence on this 3rd party action and verified it is safe to use and not at risk of malicious intent?

Reply:
Yep I had a look and it's doing exactly what I'd expect it to do

Reply:
actually had another look and found this which is done by a company and also supports to ensure only one of the above labels is set - will swap it out https://github.com/agilepathway/label-checker